HW11/hw_11_task_1.py

```diff
@@ -0,0 +1,74 @@
+# 1. Додати обробку помилок до коду з завдання №7 про телефонну книгу (тема: “Колекції та структури даних. Part 1”)
+# Повинно бути як мінімум два блоки try except, де їх використовувати — вирішуєте самі.
+
+phone_book = [{"name": "Stepan Bandera", "phone": "+380501112233"},
+              {"name": "Taras Schevchenko", "phone": "+380672223344"},
+              {"name": "Yevhen Konovalets", "phone": "+380663332255"},
+              {"name": "Lina Kostenko", "phone": "+380634441166"},
+              {"name": "Bohdan Khmelnytskii", "phone": "+380675556677"}]
+
+
+def com_stats(phone_book):
+    """ function for calculation number of elements """
+    elements_number = len(phone_book)
+    # return count of phone numbers
+    return elements_number
+
+def com_add(phone_book):
+    # try... except... else... to check if user enters required keys"""
+    try:
+        # add a new contact to the phone book
+        new_contact = input(f"Please, enter name, surname and phone number (use space between): ")
```

Comment:
nit: if there are no variables in string - `f `is not required
```suggestion
        new_contact = input("Please, enter name, surname and phone number (use space between): ")
```